File: FacturacionBackDigitalWare/Base.Datos/Contexto/Entidades/Factura.cs
namespace Base.Datos.Contexto.Entidades
{
    using System;
    using System.Collections.Generic;
    public partial class Factura
    {
        public int IdFactura { get; set; }
        public int? IdCliente { get; set; }
        public DateTime? Fecha { get; set; }

        public virtual Cliente IdClienteNavigation { get; set; }
        public virtual ICollection<DetalleFactura> DetalleFactura { get; set; }
    }
}

File: FacturacionFrontDigitalWare/src/app/pages/facturas/models/Factura.ts
import { DetalleFactura } from './DetalleFactura';
export interface Factura {
  idFactura: number,
  idCliente: number,
  nombreCliente: string,
  fecha: Date,
  cantidadProductos: number,
  valorTotal: number,
  listaDetalle: DetalleFactura[]
}
